### PR TITLE
Updated and added 7 build-scripts for packages. 

### PR DESCRIPTION
**Pull request by varsha-aaynure**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+PACKAGE_URL=https://github.com/babel/babel.git
+PACKAGE_VERSION="${1:-v7.4.0}"
+
+yum update -y 
+
+#Install required files
+yum install -y git 
+
+#To install node 12 
+wget "https://nodejs.org/dist/v12.22.4/node-v12.22.4-linux-ppc64le.tar.gz"
```
Comment (kandarpamalipeddi) [inline b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh:29]: `wget` needs to be installed before its usage.
please add `wget` to `yum install` list

or

If you can do it with `curl`, it is always appreciated.

File: j/jackson-dataformat-csv/jackson-dataformat-csv_ubi8_3.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/FasterXML/jackson-dataformats-text.git
+PACKAGE_VERSION=${1:-jackson-dataformats-text-2.10.5}
+
+echo "Usage: $0 [<PACKAGE_VERSION>]"
+echo "PACKAGE_VERSION is an optional parameter whose default value is jackson-dataformats-text-2.10.5, not all versions are supported."
+
+yum update -y
```
Comment (kandarpamalipeddi) [inline j/jackson-dataformat-csv/jackson-dataformat-csv_ubi8_3.sh:26]: Do we need `yum update` ?
If it works without update, please skip this step.

Note : Its a generic comment, please implement/apply it for all the build scripts.

File: j/jmxtrans/jmxtrans_ubi8_3.sh [hunk 3]
```diff
@@ -14,18 +14,16 @@
 #             contact "Maintainer" of this script.
 #
 # ----------------------------------------------------------------------------
-
 #!/bin/bash
 
 #Variables
 PACKAGE_URL=https://github.com/jmxtrans/jmxtrans.git
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
 PACKAGE_VERSION=jmxtrans-parent-272
```
Comment (kandarpamalipeddi) [inline j/jmxtrans/jmxtrans_ubi8_3.sh:22]: Replace this with 
`PACKAGE_VERSION="${1:-jmxtrans-parent-272}"`

File: j/jackson-module-scala/jackson-module-scala_ubi8_3.sh [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/FasterXML/jackson-module-scala.git
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
+PACKAGE_VERSION=jackson-module-scala-2.10.5
```
Comment (kandarpamalipeddi) [inline j/jackson-module-scala/jackson-module-scala_ubi8_3.sh:22]: Update these 2 lines..

`PACKAGE_VERSION="${1:-jackson-module-scala-2.10.5}"`

File: b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/babel/babel.git
+PACKAGE_VERSION="${1:-v7.4.0}"
+
+yum update -y
```
Comment (kandarpamalipeddi) [inline b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh:23]: Please avoid `yum update`.
Do it only if it is required.

File: j/jmx_prometheus_javaagent/jmx_prometheus_javaagent_ubi8_3.sh [hunk 6]
```diff
@@ -14,18 +14,16 @@
 #             contact "Maintainer" of this script.
 #
 # ----------------------------------------------------------------------------
-
 #!/bin/bash
 
 #Variables
 PACKAGE_URL=https://github.com/prometheus/jmx_exporter.git
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
 PACKAGE_VERSION=parent-0.14.0
```
Comment (kandarpamalipeddi) [inline j/jmx_prometheus_javaagent/jmx_prometheus_javaagent_ubi8_3.sh:22]: Replace these 2 lines as below :

`PACKAGE_VERSION="${1:-parent-0.14.0}"`

File: l/log4j-slf4j-impl/log4j-slf4j-impl_ubi8_3.sh [hunk 7]
```diff
@@ -14,19 +14,16 @@
...
 #
 # ----------------------------------------------------------------------------
-
-
 #!/bin/bash
 
 #Variables
 PACKAGE_URL=https://github.com/apache/logging-log4j2.git
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
 PACKAGE_VERSION=log4j-2.13.2
```
Comment (kandarpamalipeddi) [inline l/log4j-slf4j-impl/log4j-slf4j-impl_ubi8_3.sh:22]: Update these 2 line.

File: b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh [hunk 8]
```diff
@@ -0,0 +1,44 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : babel-plugin-transform-runtime
+# Version       : v7.4.0
+# Source repo   : https://github.com/babel/babel
+# Tested on     : UBI 8.3
+# Script License: Apache-2.0 License    
+# Maintainer    : Varsha Aaynure <Varsha.Aaynure@ibm.com>
```
Comment (kandarpamalipeddi) [inline b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh:8]: Add `Language` field to the Header.

```
# Maintainer : Varsha Aaynure <Varsha.Aaynure@ibm.com>
# Language : Node
```

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh (hunk 1 above)
Comment (varsha-aaynure) [reply b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh:29]: okay will add wget to yum install list.

File: j/jackson-dataformat-csv/jackson-dataformat-csv_ubi8_3.sh (hunk 2 above)
Comment (varsha-aaynure) [reply j/jackson-dataformat-csv/jackson-dataformat-csv_ubi8_3.sh:26]: Yes, sure will apply it for all these build-scripts.

File: j/jmxtrans/jmxtrans_ubi8_3.sh (hunk 3 above)
Comment (varsha-aaynure) [reply j/jmxtrans/jmxtrans_ubi8_3.sh:22]: Okay.

File: b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh (hunk 5 above)
Comment (varsha-aaynure) [reply b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh:23]: okay.

File: j/jackson-module-scala/jackson-module-scala_ubi8_3.sh (hunk 4 above)
Comment (varsha-aaynure) [reply j/jackson-module-scala/jackson-module-scala_ubi8_3.sh:22]: okay.

File: j/jmx_prometheus_javaagent/jmx_prometheus_javaagent_ubi8_3.sh (hunk 6 above)
Comment (varsha-aaynure) [reply j/jmx_prometheus_javaagent/jmx_prometheus_javaagent_ubi8_3.sh:22]: okay.

File: b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh (hunk 8 above)
Comment (varsha-aaynure) [reply b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh:8]: okay.

**Pull request closed without merging**